storage-blobs-python-quickstart-v4/README.md

```diff
@@ -0,0 +1,53 @@
+Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
+the rights to use your contribution. For details, visit https://cla.microsoft.com.
+
+When you submit a pull request, a CLA-bot will automatically determine whether you need to provide
+a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions
+provided by the bot. You will only need to do this once across all repos using our CLA.
+
+This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
+For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
+contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+
+# How to run this project
+
+## Prerequisites
+
+To complete this quickstart: 
+
+* Install [Azure Storage SDK for Python](https://github.com/Azure/azure-storage-python).
+<!-- (https://azure.microsoft.com/en-us/develop/python/)
+which makes it easy to consume Microsoft Azure Storage services.  -->
+
+If you don't have an Azure subscription, create a [free account](https://azure.microsoft.com/free/?WT.mc_id=A261C142F) before you begin.
+
+## Create a storage account using the Azure portal
+
+First, create a new general-purpose storage account to use for this quickstart. 
+
+1. Go to the [Azure portal](https://portal.azure.com) and log in using your Azure account. 
+2. On the Hub menu, select **New** > **Storage** > **Storage account - blob, file, table, queue**. 
+3. Enter a name for your storage account. The name must be between 3 and 24 characters in length and may contain numbers and lowercase letters only. It must also be unique.
+4. Set `Deployment model` to **Resource manager**.
+5. Set `Account kind` to **General purpose**.
+6. Set `Performance` to **Standard**. 
+7. Set `Replication` to **Locally Redundant storage (LRS)**.
+8. Set `Storage service encryption` to **Disabled**.
+9. Set `Secure transfer required` to **Disabled**.
+10. Select your subscription. 
+11. For `resource group`, create a new one and give it a unique name. 
+12. Select the `Location` to use for your storage account.
+13. Check **Pin to dashboard** and click **Create** to create your storage account. 
+
+After your storage account is created, it is pinned to the dashboard. Click on it to open it. Under SETTINGS, click **Access keys**. Select a key and copy the CONNECTION STRING to the clipboard, then paste it into Notepad for later use.
+
+## Configure your storage connection string
+In the application, you must provide your storage account name and account key to create a BlockBlobService object. Open the `example.py` file from the Solution Explorer in your IDE. Replace **accountname** and **accountkey** with your account name and key. 
+
+```python
+block_blob_service = BlobServiceClient.from_connection_string(conn_str="Connection string")
```

Comment:
we might want to rename the variable `block_blob_service` to `blob_service` or `blob_service_client` . This is not block blob specific.